src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/executors/MessageReadinessExecutor.java

```diff
@@ -32,8 +32,8 @@ public class MessageReadinessExecutor {
     @PostConstruct
     public void start() {
         log.info("Starting message readiness executor");
-        messageReadinessExecutorService.scheduleWithFixedDelay(messageReadinessConsumer, 5000, pollInterval,
-            TimeUnit.MILLISECONDS);
+        messageReadinessExecutorService.scheduleWithFixedDelay(messageReadinessConsumer, 9000, pollInterval,
+                                                                   TimeUnit.MILLISECONDS);
     }
```

Comment:
Maybe put an info log to say that the thread has successfully been started?

Reply:
Added now.